docs/bcd-sync.md

```diff
@@ -0,0 +1,46 @@
+# BCD synchronization
+
+The [web-platform-dx/web-features](https://github.com/web-platform-dx/web-features/) and the [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data/) repositories depend on, and complement each other. This document describes how the two projects get synchronized and how BCD and web-features maintainers should work with both repositories. If you are a consumer of web-features data, these internal steps are mostly opaque to you. If you are a web-features maintainer, read this document to decide how to map or tag BCD keys. Note that either way is fine. If you're authoring a new feature, add the keys in web-features. If you're changing keys in BCD, make the change there. The goal is to limit the times where you have to make PRs in both repositories and this document helps you to understand how this is achieved.
+
+## BCD keys in `compat_features`
+
+The source YAML files, that define each feature in the web-features, might contain a `compat_features` field.
+
+When present, the `compat_features` field consists of a list of browser-compat-data (BCD) entry keys that make up this feature (e.g., `css.properties.background-color`).
+
+If the `compat_features` field is not present in a feature's source YAML file, the resulting `.yml.dist` file (generated by the `dist` script) will contain a `compat_features` field that contains the BCD entry keys that were tagged with `web-features:<feature-id>` in the BCD repository, if any.
+
+The `compat_features` field in a feature's source YAML file always takes precedence over tags defined in BCD.
+
+## Tagging keys in BCD
+
+One way to map keys to web-features is that keys are tagged in BCD from inception. This is more likely for small, incremental changes to existing features.
+
+For example, a typical scenario could look like this:
+
+- a new key appears in BCD (e.g., via a [Collector](https://github.com/openwebdocs/mdn-bcd-collector) PR).
+- The author of the BCD PR knew which feature the key belongs to (e.g., due to a key being renamed) and tags the key with the correct `web-features:*` tag.
+- A few days later, that data is released by BCD. That triggers the next `@mdn/browser-compat-data` Dependabot upgrade PR, where we regenerate web-features data based on the updated tag and remove the `compat_features` list from the YAML file (if the list and tagged keys match).
```

Comment:
There are two repositories, it seems worth being always explicit on where PRs get created.
Also, the `compat-features` list is removed from which YAML file? The "source YAML file" or the "resulting YAML file"? Probably both as a build likely happens, but I suppose we're mostly interested in the source files here, and the sentence is not clear on whether these get updated.

```suggestion
- A few days later, that data is released by BCD. That triggers the next `@mdn/browser-compat-data` Dependabot upgrade PR in the web-features repository, where we regenerate web-features data based on the updated tag and remove the `compat_features` list from the source YAML file (if the list and tagged keys match).
```